webcompat/static/js/lib/wizard/tests/helpers/babel-register.js

```diff
@@ -0,0 +1,4 @@
+require("@babel/register")({
+  babelrc: false,
+  presets: ["@babel/env"],
+});
```

Comment:
Here babel register doesn't use default `.babelrc` and transpiles everything (similarly to webpack config for es5). This allows to use es6 modules in unit tests in node environment